Lib/test/test_difflib.py

```diff
@@ -466,13 +466,33 @@ def _assert_type_error(self, msg, generator, *args):
             list(generator(*args))
         self.assertEqual(msg, str(ctx.exception))
 
+class TestJunkAPIs(unittest.TestCase):
+    def test_is_line_junk_true(self):
+        for line in ['#', '  ', ' #', '# ', ' # ', '']:
+            self.assertTrue(difflib.IS_LINE_JUNK(line), 'should be junk: {}'.format(line))
+
+    def test_is_line_junk_false(self):
+        for line in ['##', ' ##', '## ', 'abc ', 'abc #', 'abc # ', 'Mr. Moose is up!']:
+            self.assertFalse(difflib.IS_LINE_JUNK(line), 'should not be junk: {}'.format(line))
```

Comment:
The length of this line exceeds the 79 character limit allowed by [PEP 8](https://www.python.org/dev/peps/pep-0008/#maximum-line-length).

And since testing strings contain spaces it is hard to distinguish the following cases:
```
should not be junk: abc #
```
and
```
should not be junk: abc # 
```
This is why I suggested `repr()`.

Reply:
> The length of this line exceeds the 79 character limit allowed by PEP 8.

Didn't realize this was being enforced, there are other lines over this limit as well. Will do.

> This is why I suggested `repr()`.

Got it, I misread that as rept earlier and couldn't understand what you meant.